src/category_theory/abelian/opposite.lean

```diff
@@ -29,4 +31,52 @@ instance : abelian Cᵒᵖ :=
   normal_epi := λ X Y f m, by exactI
     normal_epi_of_normal_mono_unop _ (abelian.normal_mono f.unop), }
 
+section
+
+variables {X Y : C} (f : X ⟶ Y)
+
+-- TODO: Generalize (this will work whenever f has a cokernel)
```

Comment:
Does it work to just repeat this proof with the weakened assumption?

Reply:
Yes, but the more annoying part is that we don't have the fact that when `f` has a kernel, then `f.op` has a cokernel (and vice versa).

Reply:
I see, someone should make a PR for that ;)

Reply:
But it's even more annoying than you think... I don't think we have the fact that `C\op`has zero morphisms given `C` does. 
If it was just one layer down, I would have done it here ;-)

Reply:
Fair enough, since there's a TODO for this I think it's fine as-is!